cluster/up.sh

```diff
@@ -18,7 +18,7 @@ set -ex pipefail
 
 export DEPLOY_CNAO=${DEPLOY_CNAO:-true}
 export DEPLOY_KUBEVIRT=${DEPLOY_KUBEVIRT:-true}
-export KUBEVIRT_PSA=${KUBEVIRT_PSA:-false}
+export KUBEVIRT_PSA=${KUBEVIRT_PSA:-true}
```

Comment:
It deploys kubevirt v0.59.0 (line 26)
does it support PSA already ?
(well i guess otherwise it would fail isn't it ?)